components/Card/Card.tsx

```diff
@@ -4,27 +4,49 @@ import styles from "./Card.module.css"
 import { CardListItems } from "./CardListItem"
 import { CardTitle } from "./CardTitle"
 import { SeeMore } from "./SeeMore"
+import { LabeledIcon } from "../shared"
+import styled from "styled-components"
+import { Col, Container, Row } from "../bootstrap"
 
 interface CardItem {
   billName: string
   billDescription: string
 }
 
+interface CardObject {
+  id: number
+  Name: string
+  sponsorType: string
+}
+
 interface CardProps {
   header: string | undefined
   imgSrc?: string | undefined
   subheader?: string | undefined
   bodyText?: string | undefined
   timestamp?: string | undefined
   cardItems?: CardItem[] | undefined
+  cardObjects?: CardObject[][] | undefined
```

Comment:
We added a new `items` prop in a recent PR, that allows passing in an array of elements. i think we can use that to reduce some of the duplication here.

Check out [OrgPriorityCard](https://github.com/codeforboston/maple/blob/main/stories/billDetail/OrgPriorityCard.tsx#L60) for an example of creating the items array.